concrete/single_pages/dashboard/system/registration/deactivation.php

```diff
@@ -30,17 +30,47 @@
                         </label>
                     </div>
                 </div>
-                <div class="form-group" data-group="user-deactivation-days">
-                    <label class="control-label"><?=t('Threshold')?></label>
-                    <div class="form-inline">
-                        <p><?=t('Deactivate users when they have not logged in for %s days',
-                            $form->number('userDeactivationDays', $userDeactivationDays, ['style' => 'width: 100px'])
-                        )?></p>
+                <div class="well" data-group="user-deactivation-days">
+                    <div class="form-group">
+                        <label class="control-label"><?=t('Threshold')?></label>
+                        <div class="form-inline">
+                            <p><?=t('Deactivate users when they have not logged in for %s days',
+                                $form->number('userDeactivationDays', $userDeactivationDays, ['style' => 'width: 100px'])
+                            )?></p>
+                        </div>
+                    </div>
+
+                    <div class="help-block">
+                        <?=t('Note: You will need to run the "Deactivate Users" job in order for these deactivations to actually occur.')?>
+                    </div>
+                </div>
+
+                <div class="form-group">
+                    <div class="checkbox">
+                        <label>
+                            <?=$form->checkbox('enableLogoutDeactivation', 1, $enableLogoutDeactivation)?>
```

Comment:
I'd show a big red warning if this option is checked, with a message like this:

```
WARNING: even if this option can make your site safer, it could let attackers lock users.
Please be sure to know how to re-activate blocked users (even the administrator one) before activating this option.
```

Reply:
Let's revisit the messaging on this page in a different pull request. This page should probably just have a generalized message.